config/locales/en/placements/schools/school_contacts.yml

```diff
@@ -2,29 +2,11 @@ en:
   placements:
     schools:
       school_contacts:
-        edit:
-          page_title: Placement contact
-          page_title_with_error: Error - Placement contact
-          continue: Continue
-          cancel: Cancel
-        update:
-          school_contact_updated: Placement contact updated
         remove:
           page_title: Are you sure you want to remove this placement contact? - %{school_contact_name}
           are_you_sure: Are you sure you want to remove this placement contact?
           cancel: Cancel
           remove_school_contact: Remove placement contact
         destroy:
           school_contact_removed: Placement contact removed
```

Comment:
Presumably these translations can be removed, too? It's not possible to remove a school contact.

I expect the entire file can be deleted.